sdk/python/lib/pulumi/runtime/settings.py

```diff
@@ -31,6 +31,10 @@
 _GRPC_CHANNEL_OPTIONS = [('grpc.max_receive_message_length', _MAX_RPC_MESSAGE_SIZE)]
 
 
+# excessive_debug_output enables, well, pretty excessive debug output pertaining to resources and properties.
+excessive_debug_output = False
```

Comment:
This is equivalent to what we have in the Node.js SDK:

https://github.com/pulumi/pulumi/blob/7820d2c17b40ca2345f4215b3a6789744e2784e6/sdk/nodejs/runtime/settings.ts#L32-L35

Reply:
an env var might be easier for users, but this is what we do in node 🤷